requirements.txt

```diff
@@ -2,7 +2,7 @@ anyconfig~=0.10.0
 cachetools~=4.1
 click<8.0
 cookiecutter~=1.7.0
-dynaconf<3.1.6 # Pinned because Dynaconf broke a method signature in 3.1.6 used in Kedro
+dynaconf~=3.1.2
```

Comment:
I use `3.1.2` here because some users in an attempt to fix their pipeline before might have pinned it to this version.

Reply:
Since it's a stable package, can we be a little more liberal with the boundaries? Like >=3.1.2, <4.0? ~3.1.2 will limit it to the 3.1.x series. Can't remember what we had before, off the top of my head.

Reply:
We had `~=3.1.2`. I will change it to be more liberal.